src/components/admin/models/models.ts

```diff
@@ -0,0 +1,25 @@
+/* Enums */
+export enum CategoryTypes {
+  Top = 'Category',
```

Comment:
dup to you since this is just based on preference butttttt if you want you can do enums in caps, since they're constants and I think thats how Java does it i.e. TOP = 'Category'

but I've seen both done so both are ok LOL